### DISCUSSION TITLE
What it's HXCPP_VARS and how do I download it?

### DISCUSSION THREAD

**Discussion by Chidubemmo** [Verwex/Funkin-Mic-d-Up-SC#110] [Q&A]
When using the "lime rebuild extension-webm windows" and the "lime rebuild systools windows" it alway apperead this message _"Warning: Could not find environment variables for Visual Studio
Missing HXCPP_VARS
Error: Could not automatically setup MSVC"_
What does it mean and how do I solve it

**Comment by ghost** [comment]
https://download.visualstudio.microsoft.com/download/pr/67f7bfaa-2635-43d9-ba82-26564f458881/bb057808eeff7d02561cab636b32397e298c3e770ed278602c46d119fa2e4ee8/vs_BuildTools.exe

download and install this
